### DISCUSSION TITLE
Sub button background color

### DISCUSSION THREAD

**Discussion by FluGanz** [Clooos/Bubble-Card#744] [Questions about config, custom styles and templates]
![Skärmbild 2024-08-21 125904](https://github.com/user-attachments/assets/136979af-a013-42f6-acba-e7a58e23cd56)
![Skärmbild 2024-08-21 125928](https://github.com/user-attachments/assets/7aa72615-4af9-40e5-ae32-7f3f39748caf)
When the awning is "closed" (not out) it's on, and that will make the sub button become orange. Is there a way to reverse so that when the entity is off it will become orange?

**Comment by Clooos** [comment]
Hi, yes you can, take a look at the Styling and the Templates sections in the documentation, I've also added a lot of examples.

**Reply by FluGanz** [reply]
How do i do so that it would follow light/dark system theme on ios?

**Comment by MrBearPresident** [comment]
Have you solved this?
Can this be closed?
